packages/mpt/test/trie/prune.spec.ts

```diff
@@ -135,21 +135,21 @@ describe('Pruned trie tests', () => {
         }
       }
 
-      assert.ok(await trie.verifyPrunedIntegrity(), 'trie is correctly pruned')
+      assert.isTrue(await trie.verifyPrunedIntegrity(), 'trie is correctly pruned')
 
       // Delete all keys
       for (let idx = 0; idx < 100; idx++) {
         await trie.del(keys[idx])
       }
 
-      assert.ok(await trie.verifyPrunedIntegrity(), 'trie is correctly pruned')
+      assert.isTrue(await trie.verifyPrunedIntegrity(), 'trie is correctly pruned')
       assert.isTrue(equalsBytes(trie.root(), KECCAK256_RLP), 'trie is empty')
 
       let dbKeys = 0
       for (const _dbkey of (<any>trie)._db.db._database.keys()) {
         dbKeys++
       }
-      assert.ok(dbKeys === 0, 'db is empty')
+      assert.equal(dbKeys, 0, 'db is empty')
```

Comment:
Maybe here do the `assert.isEmpty((<any>trie)._db.db._database.keys())` (remove for loop to count keys)